src/Assembly.js

```diff
@@ -64,10 +74,12 @@ class Assembly extends React.Component {
   @observable noteTitle = null
 
   // Current medication report
-  @observable survey_date = null
-  @observable survey_medication_time = null
-  @observable survey_tookMedication = null
-  @observable survey_notTakingMedicationReason = null
+  @observable survey = {
+    date: null,
+    medication_time: null,
+    not_taking_medication_reason: null,
+    took_medication: null,
+  }
```

Comment:
Important - this `survey` data structure holds all of the information about the medication report.

Reply:
I had to restructure the information in this Assembly file to more closely match our translations.
Anywhere that we use the `field` helper, the "tag" for that field represents the translation location,
as well as the location in the Assembly data structure.

Less to remember, all around.